br/pkg/restore/rawkv_client.go

```diff
@@ -31,14 +32,25 @@ func NewRawkvClient(ctx context.Context, pdAddrs []string, security config.Secur
 		pd.WithMaxErrorRetry(5))
 }
 
+func bytes2String(b []byte) string {
```

Comment:
Maybe `hack.String`? (The shape of `string` and `[]byte` isn't identity, I'm afraid convert them directly would be invalid in the future.)

https://github.com/pingcap/tidb/blob/36c348899fc9d82db8760ad1ddf33401fb7ecf14/util/hack/hack.go#L28-L37

Reply:
ok, I'll try it